kubespawner/spawner.py

```diff
@@ -33,11 +33,6 @@
 class PodReflector(NamespacedResourceReflector):
     kind = 'pods'
 
-    labels = {
-        'heritage': 'jupyterhub',
-        'component': 'singleuser-server',
-    }
-
```

Comment:
Heritage were already defined in `_build_common_labels` and `component` was added specifically anyhow in `get_pvc_manifest` and `_build_pod_labels`.

Reply:
Oh... But perhaps they were utilized by something outside the kubespawner project that I didn't know of? I did not find an internal reference to them though.

Reply:
The `labels` field in the `PodReflector` overrides the `NamespacedResourceReflector` field and I didn't realize that fully until recently. As they fill the purpose of selecting the pods within the namespace with those labels, they must remain.